### PR TITLE
[18.0] [MIG] auth_admin_passkey: Migration to 18.0

### PR DESCRIPTION
**Pull request by moncefarajdal**

### CODE REVIEW

File: auth_admin_passkey/models/res_users.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            "System Administrator user used his passkey to login"
+            " with %(login)s."
+            "\n\n\n\n"
+            "Technicals informations belows : \n\n"
+            "- Login date : %(login_date)s\n\n"
+        ) % {
+            "login": login_user.login,
+            "login_date": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
+        }
+        return subject, "<pre>{}</pre>".format(body)
```
Comment (legalsylvain) [inline auth_admin_passkey/models/res_users.py]:
```suggestion
        return subject, f"<pre>{body}</pre>"

```
nitpicking : looks lighter.

File: auth_admin_passkey/models/res_users.py [hunk 2]
```diff
@@ -53,7 +53,9 @@ def _prepare_email_passkey(self, login_user):
             "login": login_user.login,
             "login_date": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
         }
-        return subject, "<pre>{}</pre>".format(body)
+        return subject, f"<pre>{body}</pre>"`
+
+looks lighter.
```
Comment (legalsylvain) [inline auth_admin_passkey/models/res_users.py]: sorry. something has been wrong in the github proposal !

Comment (moncefarajdal) [reply auth_admin_passkey/models/res_users.py]: @legalsylvain I fixed this issue. Now I have the following issue:
AttributeError: 'dict' object has no attribute 'encode' on line:
password = hashlib.sha512(password.encode()).hexdigest()

Comment (legalsylvain) [reply auth_admin_passkey/models/res_users.py]: it looks quite unrelated, don't you think ?

Comment (moncefarajdal) [reply auth_admin_passkey/models/res_users.py]: @legalsylvain Yes your solution fixed the previous problem. It's not related to this one.

**Comment by BT-dlagin** [comment]
@moncefarajdal hello, need to migrate this module and found out this Closed PR.
Any chance of finishing the migration?
Thanks!

**Pull request closed without merging**